terratest/geo/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
```

Comment:
`main.go` looks strange in the tests. How is it getting used/executed ?

Reply:
@ytsarev it is used to generate test DB

Reply:
run `go run main.go` to get `geoip.mmdb`

Reply:
@k0da where is it executed?

Reply:
we ship compiled geoip.mmdb as part of this commit main.go is there just for refference how to generate such files.

It is referenced here https://github.com/k8gb-io/coredns-crd-plugin/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52

and blob itself here:https://github.com/k8gb-io/coredns-crd-plugin/pull/14/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R69

Reply:
please document the process